src/options.ts

```diff
@@ -29,6 +29,19 @@ function parsePath(v: string) {
   return path;
 }
 
+function parseNamespaces(v: string) {
+  const namespaces = v.split(',');
+  if (!namespaces.length) {
+    throw new InvalidArgumentError(`Empty list of namespaces.`);
+  }
+  if (namespaces.some((n) => !n)) {
+    throw new InvalidArgumentError(
+      `List of namespaces contains one or more empty entries.`
+    );
+  }
+  return namespaces;
+}
```

Comment:
This parser shouldn't be necessary; see https://github.com/tolgee/tolgee-cli/blob/f7a19f970206c77098f9ec530c3cecb4a6330584/src/commands/pull.ts#L66-L69 for example

Reply:
I didn't know about the "option" thing... I will fix this.

Reply:
Regarding the `filterExtractionResult` and your suggestion, let me share my thoughts:
Actually, I was trying to solve 2 issues here, but maybe we should separate them:
1. My entire Vue project is using a single namespace, and since the extractor is file-based, it does not detect the "default namespace" defined somewhere in the project i18n initialization.
2. My Tolgee project has other namespaces, used by other (non-Vue) projects in my monorepo. So I don't want the `sync` command to tell me there are X unused strings that can be deleted.

I therefore suggest the following changes:
- Add another option, `--default-namespace` (optional).
- Change the behavior of `filterExtractionResult` to fallback to the default namespace, instead of the first specified namespace (as in my current PR), but only if the `--default-namespace` was specified. Otherwise, it will fallback to `NullNamespace`, like in the original code.

What do you think?

Reply:
I figured the intent was to have it serve as a default namespace, yeah. I'm quite unsure what's the best solution to deal with this; having an explicit `default-namespace` seems like a good compromise. It would probably be good to also be able to define it via `tolgeerc`, to avoid having to specify it every single time.

The change to `filterExtractionResult` makes sense. I still stand by my original comment that I think it's not necessary nor desirable to have this function drop keys that are not in the namespaces specified in `--namespaces`. Since they'll be keyed by namespace after this method it's trivial to drop them afterwards, and I still think it should be reported (as it is a symptom of a potential problem with the keys)

Excluding unrelated remote keys from being included in `sync` is absolutely a desirable behavior, especially considering it makes it impossible to benefit from automatic unused key trimming otherwise.